src/css/application.css

```diff
@@ -131,73 +134,107 @@ body {
   background-color: var(--color-red);
 }
 
-.top-bar__logo-container > svg {
-  height: 3em;
+.top-bar__logo-container {
+  margin: 0 0.4em 0 0.2em;
+  min-width: 1.6em;
+  transform: scale(1.8);
+  width: 1.6em;
 }
 
 .top-bar__wordmark-container {
   position: relative;
-  margin-left: -0.25em;
+  margin: 0 1em 0 0.5em;
+  min-width: 6.5em;
+  width: 6.5em;
 }
 
 .top-bar__wordmark-container > svg {
-  padding: 0 1em;
-  max-height: 1em;
-  max-width: 100%;
-  width: auto;
+  width: 100%;
+  height: 100%;
 }
 
 .top-bar__snapshot_in-progress {
-  color: var(--color-light-gray);
+  opacity: 0.5;
   cursor: progress;
 }
 
 .top-bar__avatar {
-  height: 2em;
-  border-radius: 50%;
+  border-radius: 0.2em;
+  height: 1.8em;
   vertical-align: middle;
-  line-height: 1em;
 }
 
 .top-bar__username {
   margin-left: 0.5em;
 }
 
 .top-bar__menu-button {
-  align-self: stretch;
-  border-color: transparent;
-  border-style: solid;
-  border-width: 0 1px;
+  align-self: center;
+  border-radius: 0.2em;
   box-sizing: border-box;
   cursor: pointer;
-  line-height: 3em;
-  padding: 0 0.5em;
+  height: 2.8em;
+  line-height: 2.8em;
+  padding: 0 1em;
   position: relative;
   user-select: none;
   vertical-align: middle;
+  margin-right: 0.5em;
+  white-space: nowrap;
 }
 
-.top-bar__menu-button_active {
-  background-color: white;
-  border-color: var(--color-low-contrast-gray);
+.top-bar__menu-button:last-child {
+  margin-right: 0;
+}
+
+.top-bar__menu-button:hover {
+  box-shadow: 0 0.1em 0.2em 0 rgba(0, 0, 0, 0.3);
+}
+
+.top-bar__menu-button_primary,
+.top-bar__menu-button_secondary {
+  box-shadow: 0 0 0.2em 0 rgba(0, 0, 0, 0.3);
+}
+
+.top-bar__menu-button_primary {
+  background-color: rgba(255, 255, 255, 0.5);
+}
+
+.top-bar__menu-button_active,
+.top-bar__menu-button_active:hover,
+.top-bar__menu-button:active {
+  box-shadow: inset 0 0.1em 0.2em 0 rgba(0, 0, 0, 0.3);
 }
 
 .top-bar__hamburger {
-  font-size: 2em;
+  font-size: 1.5em;
 }
 
 .top-bar__menu {
-  border: 1px solid var(--color-low-contrast-gray);
-  box-shadow: 2px 2px 2px var(--color-low-contrast-gray);
-  left: -1px;
+  border-radius: 0.2em;
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.25);
   position: absolute;
-  top: 100%;
+  top: 3.4em;
```

Comment:
Another advantage of using an explicit height for the top bar instead of padding is that we could go back to 100% height here. That’s very appealing because it means we don’t have the knowledge of “what is the outer height of the top bar” spread over several places in the stylesheet.

Reply:
Still doesn't quite work at 100% if you want the menus to sit below the bar, not directly below the button. I'm not partial to the way this looks.

<img width="492" alt="screenshot 2017-11-16 20 33 09" src="https://user-images.githubusercontent.com/3432105/32924961-61a0aab4-cb0d-11e7-9e10-9b97562453fd.png">